light_curves/light_curve_classifier.md

```diff
@@ -4,11 +4,11 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.15.2
+    jupytext_version: 1.16.0
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: science_demo
   language: python
-  name: python3
+  name: conda-env-science_demo-py
 ---
```

Comment:
We'll need to find a way to not add changes to these metadata in unrelated PRs. E.g. changing the kernelname will certainly trip the CI in the future.

For now, my only solution is to `git add` patches rather than full files, either on the command line or with a GUI, but I'll google around if there is a nicer solution.

Reply:
I don't know how to do this, so maybe it is something to add to your git demo for GRITS this fall.  Of course, you can always explain to me before then.  I am running these timing tests on Fornax, and the metadata changes are happening all behind the scenes.

Reply:
OK, I can certainly show how to use `git add --patch filename`, as it's a useful thing for many situations.

And also, I'm looking at whether removing all these metadata from the files is feasible. So merge away, and I'll follow-up separately.

Reply:
xref: https://github.com/nasa-fornax/fornax-demo-notebooks/issues/303